mods/doors/init.lua

```diff
@@ -452,6 +456,27 @@ doors.register("door_obsidian_glass", {
 		},
 })
 
+-- Capture mods using the old API as best as possible.
+function doors.register_door(name, def)
+	if def.only_placer_can_open then
+		def.protected = true
+	end
+	def.only_placer_can_open = nil
+
+	local i = name:find(":")
+	local modname = name:sub(1, i - 1)
+	if not def.tiles then
+		if def.protected then
+			def.tiles = {{name = "doors_door_steel.png", backface_culling = true}}
+		else
+			def.tiles = {{name = "doors_door_wood.png", backface_culling = true}}
+		end
+		minetest.log("error", modname .. " registered door \"" .. name .. "\" using deprecated API method \"doors.register_door()\" but did not provide the \"tiles\" parameter. A fallback tiledef will be used instead.")
```

Comment:
Can you break this line a bit?
Also, it's technically a warning and not an error.